warehouse/models/staging/payments/littlepay/stg_littlepay__micropayment_adjustments.sql

```diff
@@ -14,7 +14,8 @@ clean_columns AS (
         {{ trim_make_empty_string_null('amount') }} AS amount,
         {{ trim_make_empty_string_null('time_period_type') }} AS time_period_type,
         {{ safe_cast('applied', type_boolean()) }} AS applied,
-        {{ trim_make_empty_string_null('zone_ids_us') }} AS zone_ids_us,
+        {{ trim_make_empty_string_null('zone_ids_used') }} AS zone_ids_used,
+        {{ trim_make_empty_string_null('incentive_product_id') }} AS incentive_product_id,
```

Comment:
`incentive_product_id` seems to be null in 100% of our data. Do we actually get data with this column? (i.e., is it actually present in any files we're currently getting?)
<img width="850" alt="image" src="https://github.com/cal-itp/data-infra/assets/55149902/9c84e668-14e3-437d-a8d5-b384a2e50d9d">